dilation-protocol.md

```diff
@@ -24,28 +24,30 @@ version present in both side's lists is eligible for use.
 ## Leaders and Followers
 
 Each side of a Wormhole has a randomly-generated dilation `side` string (this
-is included in the `please-dilate` message, and is independent of the
```

Comment:
Why are we deleting the "-dilate" from here..? Seems more clear to me to keep it.

Reply:
See #26. It's to make the documentation match the existing code

Reply:
Ah, okay sorry I thought it had "-dilate" in the code :+1: